src/sentry/sentry_apps/api/endpoints/sentry_app_details.py

```diff
@@ -107,11 +108,16 @@ def put(self, request: Request, sentry_app) -> Response:
             assert isinstance(
                 request.user, (User, RpcUser)
             ), "User must be authenticated to update a Sentry App"
+
+            feature_list = result.get("features")
+            if feature_set := result.get("feature_set"):
```

Comment:
Should we be checking for equality of these 2 ?

Reply:
Yep! I made the options mutually exclusive. There are more edge cases handling 2 than 1.